ocaml/xapi/repository.ml

```diff
@@ -149,26 +149,69 @@ let get_proxy_params ~__context repo_name =
   | _ ->
       ("", "", "")
 
-let sync ~__context ~self ~token ~token_id =
+type client_auth_conf =
+  | CdnTokenAuthConf (* remote *) of {token: string; token_id: string}
```

Comment:
The location of the comment looks odd.

Reply:
I added there and it had been formatted, or is there a better suggested place to add the comment?